packages/odyssey-storybook/src/components/PasswordInput/PasswordInput.stories.tsx

```diff
@@ -67,13 +69,7 @@ export const Themed = (
     },
   });
 
-  return (
-    <MuiThemeProvider theme={theme}>
```

Comment:
Removing this breaks the display in _some_ ways. Not sure what is going on here:

<img width="268" alt="Screen Shot 2022-06-08 at 1 59 02 PM" src="https://user-images.githubusercontent.com/36284167/172716087-c0697f99-858f-4cff-827f-cad2fb10a11d.png">

Our blue is being applied, but not our typeface. This might be expected because we don't have a font-face declaration inside the theme yet.

Reply:
Can you change the theme to use a different primary main color to check that the decorator works?

Reply:
The screenshot here is reflecting the correct color from the main director, but the "Themed" example is broken without the wrappers.

I've added the wrappers back to protect the previous work, since that's not really the point of this PR. Normal, non-theme override stories should work fine without the wrappers.

Reply:
So in order to get the decorator to use the custom theme for this story, we can do this:
```
Themed.args = createTheme({
  palette: {
    mode: "light",
    primary: {
      main: "#3fb466",
    },
  },
});

# packages/odyssey-storybook/.storybook/components/MuiThemeDecorator.tsx
export const MuiThemeDecorator: DecoratorFn = (Story, {args: theme}) => (
  <MuiThemeProvider theme={theme}>
    <ThemeProvider theme={theme}>
      <Story />
    </ThemeProvider>
  </MuiThemeProvider>
);

Reply:
Looks like my previous comment doesn't work for when changing the color via Controls